index.js

```diff
@@ -1 +1,109 @@
-// implement your API here
+const express = require("express");
+
+const db = require("./data/db.js");
+const server = express();
+
+// MiddleWare
+server.use(express.json());
+
+// Create - Adds a new user to the db - Endpoint 'api/users'
+server.post("/api/users", (req, res) => {
+  const { name, bio } = req.body;
+  if (!name || !bio) {
+    res
+      .status(400)
+      .json({ errorMessage: "Please provide a name and bio for the user" });
+  }
+  db.insert({ name, bio })
+    .then(response => {
+      res.status(201).json(response);
+    })
+    .catch(err => {
+      res.status(500).json({
+        error: "There was an error while saving the user to the database!!!"
+      });
+    });
+});
+
+// Read - Gets the full list of users on the db - Endpoint 'api/users'
+server.get("/api/users", (req, res) => {
+  db.find()
+    .then(allUsers => {
+      res.send(allUsers);
```

Comment:
make sure to always return a status code as it lets the front end know that there are no hidden errors or warnings through the header.